core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -176,22 +176,6 @@
       "One-line configuration property controlling the network locations "
           + "(hostnames) that are allowed to impersonate other users",
       "1.7.1"),
-  // Crypto-related properties
-  @Experimental
-  INSTANCE_CRYPTO_PREFIX("instance.crypto.opts.", null, PropertyType.PREFIX,
-      "Properties related to on-disk file encryption.", "2.0.0"),
-  @Experimental
-  @Sensitive
-  INSTANCE_CRYPTO_SENSITIVE_PREFIX("instance.crypto.opts.sensitive.", null, PropertyType.PREFIX,
-      "Sensitive properties related to on-disk file encryption.", "2.0.0"),
-  @Experimental
-  INSTANCE_CRYPTO_SERVICE("instance.crypto.service",
-      "org.apache.accumulo.core.spi.crypto.NoCryptoService", PropertyType.CLASSNAME,
-      "The class which executes on-disk file encryption. The default does nothing. To enable "
-          + "encryption, replace this classname with an implementation of the"
-          + "org.apache.accumulo.core.spi.crypto.CryptoService interface.",
-      "2.0.0"),
-
```

Comment:
Part of the benefit of having crypto properties as `instance.` properties ensures that all tservers are using the same config. The potential for different tservers to have different crypto services means that data written by one tserver might not be able to be read by another, as they could have wildly different behaviors. This is one reason why I think we should keep an `instance.` property to refer to the crypto service factory, which is the overall module that supports all the different crypto that will be needed, and that per-tserver and per-table properties should only modify the behavior of the crypto services produced by this factory with some options.

Reply:
That makes sense. I had only gone with "tserver" for "wal" properties since that is where most of the write ahead log configuration lives.

Reply:
Yeah, I can see it both ways: "wal" is a tserver feature, but "crypto" support is an instance feature. I see this as somewhat more closely aligned with the way volume chooser does things, where it's specified at the system scope (using `general.` prefix) but a specific implementation could delegate to per-table config to act in a per-table way.